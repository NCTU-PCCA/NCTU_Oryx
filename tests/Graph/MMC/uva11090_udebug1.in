3
18 63
17 9 10
3 16 12
18 4 5
13 3 4
18 5 11
12 16 4
13 2 1
15 4 13
6 10 7
10 6 17
17 3 9
2 5 6
3 4 17
4 8 8
8 12 2
3 12 4
4 10 12
7 4 2
17 4 3
2 16 8
2 3 1
6 18 6
6 3 8
13 7 5
14 15 4
15 7 15
2 8 16
16 10 11
17 7 18
1 9 18
4 7 16
7 9 15
4 12 19
11 2 6
2 13 17
5 8 15
2 9 8
11 6 11
7 2 12
4 18 10
18 1 20
6 5 2
15 1 18
15 18 13
9 14 12
16 14 5
2 4 13
6 17 12
5 16 8
11 4 6
10 3 12
8 3 1
3 8 12
17 12 12
18 9 6
1 12 10
6 16 12
9 10 20
1 11 8
8 13 11
15 14 11
16 1 15
12 13 18
3 4 
1 2 2 
2 3 4 
3 1 6 
1 3 6
3 5 
1 2 1 
2 3 5 
3 1 1 
3 1 5 
2 3 1 

