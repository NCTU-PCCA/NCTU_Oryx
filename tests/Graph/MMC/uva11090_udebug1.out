Case #1: 5.38
Case #2: 4.00
Case #3: 1.00
